app/src/main/res/layout/activity_main.xml

```diff
@@ -33,14 +33,21 @@
             app:popupTheme="@style/ThemeOverlay.AppCompat.Light"
             android:elevation="4dp">
 
+            <de.westnordost.streetcomplete.statistics.UploadsCounter
+                android:id="@+id/uploadsCounter"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="end"
+                android:drawablePadding="2dp"
+                android:drawableStart="@drawable/ic_file_upload_24dp"
+                android:gravity="center_vertical"
+                />
+
             <de.westnordost.streetcomplete.statistics.AnswersCounter
                 android:id="@+id/answersCounter"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_gravity="end"
-                android:minEms="7"
-                android:paddingEnd="8dp"
-                android:paddingStart="8dp"
```

Comment:
Wouldn't that look very packed on smaller devices? Does it even fit with four-digit counters each?

I think it is not really necessary to display the app's name in the header. This way, there will be more than enough space.

Reply:
Removed the title and moved the counters to the left. I added back the padding. The minEms is adding a lot of space so not sure if its needed. The text being dynamic seems to make enough space for itself and the padding ensures they are not squished. Tested the layout on on a pixel emulator.